theory/README.md

```diff
@@ -33,6 +38,7 @@ both `antelope` and `antelopes`.)
   
   812-2-10
 
+  `/[0-9]/g`
```

Comment:
Since we're specifically matching the pattern YYYY-MM-DD, something like this can work: `/\d{1,4}-\d{1,2}-\d{1,2}/g`